pm2.config.cjs

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Can you please explain why you are using a .cjs file instead of a plain .js one?

I do not see anything relative to that in https://pm2.keymetrics.io/docs/usage/application-declaration/

Thanks

Reply:
Because if we do not use a `cjs`file we have this following error:
```
[PM2][ERROR] File pm2.config.js malformated
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/ndpnt/Workspace/OTA/OpenTermsArchive/pm2.config.js
require() of ES modules is not supported.
require() of /Users/ndpnt/Workspace/OTA/OpenTermsArchive/pm2.config.js from /Users/ndpnt/.nvm/versions/node/v14.18.0/lib/node_modules/pm2/lib/Common.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename pm2.config.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /Users/ndpnt/Workspace/OTA/OpenTermsArchive/package.json.
```